integration_tests/test_solr_search_results.py

```diff
@@ -0,0 +1,127 @@
+import pytest
+import requests
+import typing
+import os
+
+
+@pytest.fixture
+def rsession():
+    return requests.session()
+
+
+@pytest.fixture
+def solr_url():
+    solr_url = os.getenv("INPUT_SOLR_URL")
+    if solr_url is None:
+        solr_url = "https://mars.cyverse.org/thing/select"
+    return solr_url
+
+
+def headers():
+    return {
+        "accept": "application/json",
+        "User-Agent": "iSamples SOLR Integration Bot 2000",
+    }
+
+
+authority_ids = [
+    "SESAR",
+    "OPENCONTEXT",
+    "GEOME",
+    "SMITHSONIAN",
+]
+
+
+def _send_solr_query(
+    rsession: requests.sessions, solr_url: str, query: str
+) -> typing.List[typing.Dict]:
+    params = {
+        "start": 0,
+        "limit": 10,
+        "fl": "*",
+        "q": f"searchText:({query})",
```

Comment:
@datadavev, after getting some unexpected results, I changed the solr query to be parenthesized and that caused us to get better results in a couple cases (New Zealand, United States).  Let me know if you think that is a reasonable default.

Reply:
Though even with the parentheses, "Excavations at Polis" still didn't work as expected.